src/pages/Main/Recommendation/Recommendation.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import RecommendedBook from './RecommendedBook';
+import { AiOutlineDoubleLeft } from 'react-icons/ai';
+import { AiOutlineDoubleRight } from 'react-icons/ai';
```

Comment:
```suggestion
import { AiOutlineDoubleLeft, AiOutlineDoubleRight } from 'react-icons/ai';
```
임포트를 하실 때도 결국 객체의 형태를 활용할 수 있습니다!

Reply:
알려주신 바와 같이 수정했습니다!